src/common-utils/main.sh

```diff
@@ -104,18 +104,22 @@ install_debian_packages() {
 
     # Needed for adding manpages-posix and manpages-posix-dev which are non-free packages in Debian
```

Comment:
Can we update patch version of this Feature? https://github.com/devcontainers/features/blob/e5054dba652b09902ec43aa45bf05386a28a341c/src/common-utils/devcontainer-feature.json#L3

Reply:
done 👍